File: MPI/mpicheck/tests/functions/2.0/MPI_File_open.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_file_open(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_file_open(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_File_open.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    const char *var_1;
    int var_2;
    MPI_Info var_3;
    MPI_File *var_4;
    int ret;
    /* calls */
    ret = MPI_File_open(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_File_open(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
